charts/kong/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 2.42.0
+
+### Changes
+
+* Bumped default `kong` image tag to 3.7.
```

Comment:
Or maybe we should go with [3.8](https://konghq.com/blog/product-releases/kong-gateway-3-8)?
See also https://github.com/Kong/charts/issues/1130

Reply:
ACK #1130.

Let's have at at least 1 version with 3.7 to allow users to gradually migrate.